README.md

```diff
@@ -78,7 +78,7 @@ the headline section (right after the header):
 
 > Add a link to your deployed project.
 
-- [Live Demo Link]( https://princebadru.github.io/mobilefirst/)
+- [Live Demo Link](https://princebadru.github.io/mobilefirst/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I have realised that the following three changes to the README were not made. Kindly make sure that you make them before you submit for review.

![image](https://user-images.githubusercontent.com/86318284/217284603-8ec3d1d6-d859-4802-bd0d-4a594b27386d.png)